src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Decimal.cs

```diff
@@ -42,8 +42,8 @@ public static unsafe bool TryFormat(decimal value, Span<byte> destination, out i
                         if (format.Precision != StandardFormat.NoPrecision)
                             throw new NotSupportedException(SR.Argument_GWithPrecisionNotSupported);
 
-                        byte* pDigits = stackalloc byte[Number.DecimalNumberBufferLength];
-                        Number.NumberBuffer number = new Number.NumberBuffer(Number.NumberBufferKind.Decimal, pDigits, Number.DecimalNumberBufferLength);
+                        Span<byte> pDigits = stackalloc byte[Number.DecimalNumberBufferLength];
```

Comment:
Nit: Using `p` prefix for spans does not look good. I would rename all Span `pDigits` to just `digits`.

Reply:
Done